src/components/ActivitiesListFilters.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {setTextFilter} from '../actions/filters';
+
+const ActivitiesListFilters = (props) => (
+  <div>
+    <input type="text" value={props.filters.text} onChange={(e) => {
+      props.dispatch(setTextFilter(e.target.value))
+    }}/>
+  </div>
+);
+
+const mapStateToProps = (state) => {
```

Comment:
es6 shorthand. 
You sould think about seting up also `eslint` and `stylelint`. Powerful and it makes our lives easier (together with `prettier`)